restaurants/views.py

```diff
@@ -69,3 +69,25 @@ def get(self, request, restaurant_id):
 
         except Restaurant.DoesNotExist:
             return JsonResponse({"message":"RESTAURANT_NOT_EXIST"}, status=404)        
+
+class RestaurantReviewView(View):
+    def get(self, request, restaurant_id):
+        offset        = int(request.GET.get("offset", 1))
+        limit         = int(request.GET.get("limit", 10))
+        rating_min    = request.GET.get("rating-min", 1)
+        rating_max    = request.GET.get("rating-max", 5)
+        reviews       = Review.objects.filter(restaurant_id=restaurant_id, rating__gte = rating_min, rating__lte = rating_max).order_by("-created_at")[offset : limit]
+        review_list   = [{
+                "user":{
+                    "id":r.user.id,
+                    "nickname":r.user.nickname,
+                    "profile_image":r.user.profile_url,
+                    "review_count":r.user.reviewed_restaurants.count()
+                },
+                "id":r.id,
+                "content" : r.content,
+                "rating":r.rating,
+                "created_at":r.created_at,
+            } for r in reviews]
```

Comment:
Align 맞춰주시고, r -> review로 바꿔주세요.

```suggestion
        review_list   = [{
            "user":{
                "id":r.user.id,
                "nickname":r.user.nickname,
                "profile_image":r.user.profile_url,
                "review_count":r.user.reviewed_restaurants.count()
                },
                "id":r.id,
                "content" : r.content,
                "rating":r.rating,
                "created_at":r.created_at,
            } for r in reviews]
```

Reply:
해당 사항 수정했습니다!

Reply:
dictionary convention 해결했습니다!